content/WebAuthn/WebAuthn_Developer_Guide/WebAuthn_Client_Registration.adoc

```diff
@@ -54,7 +54,7 @@ _Note: In the WebAuthn API, all binary parameters (`user.id`, `challenge` etc.)
 
 **attestation:** The default value is “none” which means the RP is not interested in authenticator attestation. “indirect” indicates the RP prefers a verifiable attestation statement but allows the client to decide how to obtain it. “direct” indicates the RP wants to receive the attestation statement. It is recommended that RPs use the “direct” value and store the attestation statement with the credential so they can inspect authenticator models retroactively if policy changes.
 
-**authenticatorSelection:** specify authenticator requirements. The optional **authenticatorAttachment** attribute filters eligible authenticator by type. The value “platform” indicates a platform authenticator, such as Windows Hello. The value "cross-platform" value indicates a roaming authenticator, such as a security key. When the **requireResidentKey** attribute is true, the authenticator must create a client side resident private key (a.k.a. resident credential). The **userVerification** attribute can be set to a value of “preferred”, “required”, or “discouraged”.
+**authenticatorSelection:** specify authenticator requirements. The optional **authenticatorAttachment** attribute filters eligible authenticator by type. The value “platform” indicates a platform authenticator, such as Windows Hello. The value "cross-platform" value indicates a roaming authenticator, such as a security key. When the **requireResidentKey** attribute is true, the authenticator must create a client side discoverable private key (a.k.a. discoverable credential). The **userVerification** attribute can be set to a value of “preferred”, “required”, or “discouraged”.
```

Comment:
the 'residentKey' param will supersede the 'requireResidentKey' param when the spec is released.